frontend/src/_components/Map.tsx

```
@@ -2,33 +2,49 @@
 
 import { useEffect, useRef } from "react";
 import * as Phaser from "phaser";
+import { useQuery } from "@tanstack/react-query";
+import { petApi } from "@/lib/api/pet";
 import { gameConfig } from "@/game/config";
 import { useAuthStore } from "@/stores/authStore";
 
 export default function Map() {
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)